### PR TITLE
solved problem with clear background of section header

### PR DESCRIPTION
**Pull request by Slawbel**

### CODE REVIEW

File: Sources/ConverterScreen.swift [hunk 1]
```diff
@@ -826,10 +826,10 @@ extension ConverterScreen: DiagramResultDelegate {
             cutShortNameFlag = self.getFlagToLabel(shortName: chosenCurrency!)
         }
         
-        if cutShortNameFlag == nil {
+        if chosenCurrency == nil {
```
Comment (RomanPodymov) [inline Sources/ConverterScreen.swift:829]: What about to refactor it with `if let chosenCurrency`?

File: Sources/ConverterScreen.swift [hunk 2]
```diff
@@ -747,7 +747,7 @@ class ConverterScreen: UIViewController {
     // function to prepare code for finding of flag symbol
     func getFlagToLabel (shortName: String) -> String? {
         var cutShortNameFlag: String?
-        if shortName != "BTC" && shortName != "XOF" && shortName != "XAF" && shortName != "XPF" && shortName != "STD" && shortName != "XAG" && shortName != "XAU" {
+        if shortName != "BTC" && shortName != "XOF" && shortName != "XAF" && shortName != "XPF" && shortName != "STD" && shortName != "XAG" && shortName != "XAU" && shortName != "ANG" && shortName != "XCD" && shortName != "XDR" {
```
Comment (RomanPodymov) [inline Sources/ConverterScreen.swift:750]: What about an array with `let shortNames = ["BTC", "XOF", "XAF" ...]` that you can use for this check?

**Review by RomanPodymov — CHANGES_REQUESTED** [review]

File: Sources/ConverterScreen.swift (hunk 2 above)
Comment (Slawbel) [reply Sources/ConverterScreen.swift]: Corrected

File: Sources/ConverterScreen.swift (hunk 1 above)
Comment (Slawbel) [reply Sources/ConverterScreen.swift]: Corrected

**Review by RomanPodymov — APPROVED** [review]

**Pull request merged**